packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -321,6 +331,20 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
             };
         }
 
+        this.socket.on("get_ops_response", (result) => {
+            const messages = result.messages as ISequencedDocumentMessage[];
+            if ((result.nonce as string).startsWith(this.requestOpsNoncePrefix) && messages.length > 0) {
+                this.logger.sendTelemetryEvent({
```

Comment:
Should the latency of the get_ops request be logged with this event? Could track the start time for the given nonce so it's accessible here.

Reply:
It will make code a bit more complicated. I want to get this in first, and then keep improving based on feedback.
Including getting data on if this is the actual direction we want to go. I.e. I believe we will need to make a choice of either using this flow to understand better / easier why gaps are happening and keep improving PUSH, or vice verse - experiment with simplifying PUSH toward putting complex on client side